### PR TITLE
Robby russel theme update

### LINKED ISSUE #534: Robby russell theme issue
Robby russell theme isn't showing the python venv/condavenv information even after enabling it in `.bashrc` file

### PR DESCRIPTION
**Pull request by narasimha00**
### ****Robby russell theme update****
_- Added python venv tag (env) support ._

Fixes #534

### CODE REVIEW

**Review by akinomyoga — APPROVED** [review]

File: themes/robbyrussell/robbyrussell.theme.sh [hunk 1]
```diff
@@ -3,12 +3,36 @@
...
+#
+# Example inside clean git repo:
+# ➜  config-files git:(main)
+#
+# Example inside dirty git repo:
+# ➜  config-files git:(main ?:1) ✗
+#
+# Example with virtual environment:
+# ➜  (env1) ~
+#
```
Comment (akinomyoga) [inline themes/robbyrussell/robbyrussell.theme.sh:17]: There is a trailing space here.

File: themes/robbyrussell/robbyrussell.theme.sh [hunk 2]
```diff
@@ -3,12 +3,36 @@
...
+OMB_PROMPT_VIRTUALENV_FORMAT='(%s) '
+OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:=true}
+
 function _omb_theme_PROMPT_COMMAND() {
     if [[ "$?" == 0 ]]; then
         local arrow_color="${_omb_prompt_bold_green}"
     else
         local arrow_color="${_omb_prompt_bold_brown}"
     fi
+
```
Comment (akinomyoga) [inline themes/robbyrussell/robbyrussell.theme.sh:32]: Here's another trailing space.

**Review by akinomyoga — CHANGES_REQUESTED** [review]
The CI test fails. There are two lines with trailing spaces. Could you remove them?

Also, do not recreate a new PR. You can push an additional commit to the branch. Then, the update will be automatically reflected in this PR.

**Review by narasimha00 — COMMENTED** [review]
Removed trailing spaces

**Review by akinomyoga — APPROVED** [review]

**Comment by akinomyoga** [comment]
Thanks!

**Pull request merged**